src/components/graph-view.js

```diff
@@ -1303,6 +1484,43 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     this.renderNodesTimeout = setTimeout(this.renderNodes);
   }
 
+  isNodeSelected = (node: INode) => {
+    const { selectedNodes, nodeKey } = this.props;
```

Comment:
If selectedNodes and selectedEdges are dictionaries, then these lookups would be simple. Since order is arbitrary, any reason not to do this?

Reply:
If we accept a dictionary as the selectedNodes parameter then we might be forcing the users to use Object.values, Object.keys, or `for in` every time they want to loop through the selectedNodes. An array is simpler for the end user IMO, although it might add some extra loops for us.